binding.cc

```diff
@@ -542,63 +526,46 @@ struct Iterator {
   }
 
   bool GetIterator () {
-    if (dbIterator_ == NULL) {
-      dbIterator_ = database_->NewIterator(options_);
-
-      if (start_ != NULL) {
-        dbIterator_->Seek(*start_);
-
-        if (reverse_) {
-          if (!dbIterator_->Valid()) {
-            // if it's past the last key, step back
-            dbIterator_->SeekToLast();
-          } else {
-            std::string keyStr = dbIterator_->key().ToString();
-
-            if (lt_ != NULL) {
-              // TODO make a compount if statement
-              if (lt_->compare(keyStr) <= 0) {
-                dbIterator_->Prev();
-              }
-            } else if (lte_ != NULL) {
-              // TODO make a compount if statement
-              if (lte_->compare(keyStr) < 0) {
-                dbIterator_->Prev();
-              }
-            } else if (start_ != NULL) {
-              // TODO make a compount if statement
-              if (start_->compare(keyStr)) {
-                dbIterator_->Prev();
-              }
```

Comment:
The old code, though longer, made it more obvious that lt/lte/start are exclusive options.

Reply:
Yes, let's.